.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
     permissions:
       attestations: write
-      contents: read
+      contents: write
```

Comment:
This is needed so that when a build happens for a release tag, sbom-action can attach the generated file to the release.